### PR TITLE
fix: [fileviewmodel] refresh sync

### PR DESCRIPTION
**Pull request by pppanghu77**
f5 diff rightclick, Caused by different execution speeds of threads

Log: as des
Bug: https://pms.uniontech.com/bug-view-192501.html

### CODE REVIEW

**Review by max-lvs — DISMISSED** [review]

File: src/plugins/filemanager/core/dfmplugin-workspace/utils/filesortworker.cpp
Comment (Lighto-Ku) [inline src/plugins/filemanager/core/dfmplugin-workspace/utils/filesortworker.cpp:1]: 1.从信号命名来看removeFinishResult与removeFinish不好区分，建议调整removeFinishResult为requestFetchMore。
2.handleClean函数只有刷新逻辑会调用，并且还承担了请求再次获取文件数据的职责，建议调整函数名称，明确是刷新逻辑相关的函数。

Comment (Lighto-Ku) [reply src/plugins/filemanager/core/dfmplugin-workspace/utils/filesortworker.cpp:1]: -1

Comment (pppanghu77) [reply src/plugins/filemanager/core/dfmplugin-workspace/utils/filesortworker.cpp:1]: 已修改为 requestFetchMore handleRefresh

**Review by Lighto-Ku — COMMENTED** [review]
+1

**Review by max-lvs — APPROVED** [review]

**Pull request merged**